<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>quokkagit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/git.png</normaloff>:/git.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="1">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="0" column="3">
            <widget class="QLineEdit" name="searchLineEdit"/>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="searchOptionsToolButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../resources/resources.qrc">
               <normaloff>:/search.png</normaloff>:/search.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="branchLabel">
             <property name="text">
              <string>Branch</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="tableViewToolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/resources.qrc">
            <normaloff>:/table-properties.png</normaloff>:/table-properties.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="CTableWidget" name="logTableView">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>300</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>21</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="branchSelectionToolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/resources.qrc">
            <normaloff>:/tree.png</normaloff>:/tree.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTextEdit" name="pMessageTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="CTableWidget" name="pFilesTableView">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>400</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>21</number>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_log"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/quit.png</normaloff>:/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_log">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/log.png</normaloff>:/log.png</iconset>
   </property>
   <property name="text">
    <string>Open log ...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableWidget</class>
   <extends>QTableView</extends>
   <header>src/widgets/tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>logTableView</tabstop>
  <tabstop>pMessageTextEdit</tabstop>
  <tabstop>pFilesTableView</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>searchOptionsToolButton</tabstop>
  <tabstop>branchSelectionToolButton</tabstop>
  <tabstop>tableViewToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
